public/video-ui/src/components/VideoUpload/Upload.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import {browserHistory} from 'react-router';
+import Icon from '../Icon';
+
+class VideoUpload extends React.Component {
+  getUploadPolicy = (event) => {
+    if(!this.props.video)
```

Comment:
I'm happy to add it ([keyword-spacing](http://eslint.org/docs/rules/keyword-spacing)) but would do so under a different PR since enabling it would add 27 errors to fix (not in changes I've made).